.github/workflows/docker.yml

```diff
@@ -17,15 +17,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        build_preset: ["dev", "lean", "py310", "websocket", "dockerize"]
+        build_preset: ["dev", "lean", "py311", "websocket", "dockerize"]
         platform: ["linux/amd64", "linux/arm64"]
-        exclude:
```

Comment:
I'd say we keep the exclude here, this runs for every commit in every PR, and on every merge onto master, and arm is quite slow/expensive because it's emulated with QEMU